migrations/Version202108190925463772_taoLti.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoLti\migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use oat\tao\scripts\tools\migrations\AbstractMigration;
+use oat\tao\model\search\SearchProxy;
+use oat\taoLti\models\classes\ConsumerService;
+use oat\taoLti\models\classes\ProviderService;
+
+final class Version202108190925463772_taoLti extends AbstractMigration
+{
+
+    public function getDescription(): string
+    {
+        return 'Register LTI class URI to use generic search in ' . SearchProxy::class;
+    }
+
+    public function up(Schema $schema): void
+    {
+        $generisSearchWhitelist = $this->getLTIClassURI();
+        $searchProxy = $this->getProxy();
+        
+        if ($searchProxy->hasOption(SearchProxy::OPTION_GENERIS_SEARCH_WHITELIST)) {
```

Comment:
Lets make this simpler, please: https://github.com/oat-sa/extension-tao-publishing/pull/115/files#r695439951

Another idea is to create a method in `SearchProxy`, like extend white list: 

```
public function extendGenerisSearchWhiteList(array $whiteList)
{
    $this->setOption(
          self::OPTION_GENERIS_SEARCH_WHITELIST, 
          array_merge(
              $this->getOption(self::OPTION_GENERIS_SEARCH_WHITELIST, []),
               $whiteList
         )
    );
}
```

And another to `removeFromGenerisSearchWhiteList`

Doing the opposite.